src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -55,9 +55,15 @@ const SubscriberHistory = ({ location, dependencies: { dopplerApiClient } }) =>
         if (!response.success) {
           setState({ loading: false });
         } else {
+          const sentCampaigns = response.value.items.map((campaign) => {
+            return {
+              ...campaign,
+              imgPreview: campaign.links.find((x) => x.rel === '/docs/rels/get-campaign-preview'),
```

Comment:
I am not sure if it is so simple, for example, a link could have multiple relations, for example: `"rel": "/docs/rels/get-campaign-preview preview image"`

What do you think about adding a helper for this?

```suggestion
              imgPreview: searchLinkByRel(campaign, '/docs/rels/get-campaign-preview'),
```

Reply:
What happens when there is no link with specified rel?

Reply:
we talk with @cbernat in the api the rel is this for the preview, if not dont have preview and only show the link to campaign report

Reply:
I think that @fcoronelmakingsense has this case covered, for not existing preview link.

Reply:
fixed now!